### PR TITLE
Preview for excel file/files

### PR DESCRIPTION
**Pull request by vlrsmlkv**
Realize opportunity to see all output data witout downloading

Place all the sheets in order under the control buttons. And let the general sheet be at the top.

And dont forget that preview should be the same structure as downloaded file/files

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -9,42 +9,34 @@ import {createExcelFile, downloadExcelFile, getProcessedData} from "./utils.js"
...
 
-import { ws_header } from './consts.js';
+import { ws_header } from './consts';
 
 const App = () => {
   const [fileData, setFileData] = useState(null);
   const [processedData, setProcessedData] = useState(null);
   const [checkboxState, setCheckboxState] = useState(false);
 
+  useEffect(() => setProcessedData(null), [fileData])
```
Comment (OUghTMS) [inline src/App.jsx:19]: move into onChange for fileData

File: src/components/PreviewProcessedData.jsx [hunk 2]
```diff
@@ -1,29 +1,68 @@
-// import { useState } from "react";
+import classNames from "classnames";
+import { useState } from "react";
+import { Eye } from 'react-feather';
 
-const PreviewProcessedData = ({data, header, isVisibleButtons}) => {
+const PreviewProcessedData = ({isDisabledShowButton, data, onShowClick, headerCells, isVisibleButtons}) => {
```
Comment (OUghTMS) [inline src/components/PreviewProcessedData.jsx:5]: line break

**Review by OUghTMS — CHANGES_REQUESTED** [review]

File: src/App.jsx [hunk 3]
```diff
@@ -9,43 +9,58 @@ import {createExcelFile, downloadExcelFile, getProcessedData} from "./utils.js"
...
-        <p>Шаг 2</p>
-        <button 
-          disabled={!fileData}   
-          onClick={() => setProcessedData(getProcessedData(fileData))}>
-          <span>Обработать файлы</span>
-        </button>
-        <PreviewProcessedData data={processedData}/>
+        <div>
+          <p>Шаг 3</p>
+          <button className="download-button"
```
[the remark this answers is not in the document]
Comment (OUghTMS) [reply src/App.jsx:40]: ???

[the remark this answers is not in the document]
Comment (vlrsmlkv) [reply src/App.jsx:40]: really done

File: src/components/PreviewProcessedData.jsx (hunk 2 above)
Comment (vlrsmlkv) [reply src/components/PreviewProcessedData.jsx:5]: done

File: src/App.jsx (hunk 1 above)
Comment (vlrsmlkv) [reply src/App.jsx:19]: done i guess

File: src/components/FileField.jsx [hunk 4]
```diff
@@ -10,6 +10,7 @@ const FileField = ({onChange}) => {
     
     setFilesNumber(files.length)
     setFilesNameList(files[0].name)
+    setProcessedData(null);
```
Comment (OUghTMS) [inline src/components/FileField.jsx:13]: net blat ne suda

**Review by OUghTMS — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (OUghTMS) [reply src/App.jsx]: net

File: src/components/FileField.jsx (hunk 4 above)
Comment (vlrsmlkv) [reply src/components/FileField.jsx]: fixed

File: src/App.jsx (hunk 1 above)
Comment (vlrsmlkv) [reply src/App.jsx]: fixedfixed

**Review by OUghTMS — APPROVED** [review]

**Pull request merged**